tensorflow/go/session.go

```diff
@@ -65,6 +65,53 @@ func NewSession(graph *Graph, options *SessionOptions) (*Session, error) {
 	return s, nil
 }
 
+type Device struct {
```

Comment:
Mind adding some commentary to make godoc nice?

Reply:
Done, thanks.